docs/commands/changeset-derive-replacement.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[changeset-derive-replacement]]
```

Comment:
Description for the new command, analogous to `changeset-derive` but with additional parameters